empty-s3-bucket.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# WARNING:  DESTRUCTIVE OPERATION
+
+# Recursively empties the contents of an S3 bucket.  You must be in the org and
+# space of the bucket you would like to empty.
+
+# Requires the cf CLI, awscli, and jq.
+
+set -e -x
+
+if [ "$#" -ne 1 ]; then
+  echo
+  echo "Usage:"
+  echo "   ./empty-s3-bucket.sh <service instance name>"
+  echo
+
+  exit 1
+fi
+
+# Create a service key and retrieve the credentials from it to use with awscli.
+SERVICE_INSTANCE_NAME="$1"
+KEY_NAME="$SERVICE_INSTANCE_NAME-key"
```

Comment:
The key name could conflict with an existing key. Not sure this matters.

Reply:
Good thing to call out!  I tested this and the `cf` CLI just carries on with a message saying it's already there, but still returns an `OK` - it does not cause an issue.

Reply:
I added a comment to call this out directly.